src/main/webapp/app/exercises/modeling/participate/modeling-submission.component.html

```diff
@@ -8,7 +8,7 @@
             class="guided-tour submission-button"
             id="submit-modeling-submission"
             *ngIf="isOwnerOfParticipation"
-            [disabled]="(!isActive && !isLate) || !submission || !!result || (isLate && !!submission.submitted)"
+            [disabled]="(!isActive && !isLate) || !submission || (!!result && submission?.plagiarismDetected && !isActive) || (isLate && !!submission.submitted)"
```

Comment:
Add an issue for:
1. Reducing the condition complexity.
2. Opening feedback as long as the deadline is not over yet even if a tutor posts an assesment (and not only when a case of plagiarism is detected).

Reply:
You can also add one variable containing the value of `submission?.plagiarismDetected && !isActive`, since you often reuse it

Reply:
Issue for 1: 

For 2 I suggest @MarkusPaulsen you create it yourself, as this is out of my scope and you better understand what you mean

@JohannesStoehr The issue I created will cover your comment